File: Python/RemoveDuplicates.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        # First Solution
        i = 0;
        while i + 1 < len(nums):
            if nums[i] == nums[i + 1]:
                nums.remove(nums[i])
            else:
                i = i + 1

        ##Second Solution


if __name__ == '__main__':
    solution = Solution()
    sortedarray = [0, 1, 1, 1, 1, 1, 2, 2]
    solution.removeDuplicates(sortedarray)
    print(sortedarray)

File: src/com/example/helloworld/RemoveDuplicates.java
package com.example.helloworld;

class SolutionRD {
    public int removeDuplicates(int[] nums) {
        int i = 0;
        for (int j = 0; j < nums.length; j++) {
            if (nums[j] != nums[i]) {
                i++;
                nums[i] = nums[j];
            }

        }
        return i + 1;

    }
}

public class RemoveDuplicates {
    public static void main(String[] args) {
        int[] nums = new int[]{0, 1, 1, 1, 1, 1, 2, 2};
        SolutionRD solutionRD = new SolutionRD();
        int len = solutionRD.removeDuplicates(nums);
        System.out.println(len);
    }
}
